wandb/sdk/launch/_project_spec.py

```diff
@@ -23,9 +26,13 @@
 class Project(object):
     """A project specification loaded from an MLproject file in the passed-in directory."""
 
+    dir: str
```

Comment:
is there a reason we only declare this one up here

Reply:
I need to add the rest still, but was considering a full typing PR